### PR TITLE
fix(Nettruyen): change domain

### PR DESCRIPTION
**Pull request by quanghuy38**
it changed again 💀

### CODE REVIEW

File: websites/N/Nettruyen/dist/metadata.json [hunk 1]
```diff
@@ -11,10 +11,10 @@
 		"nl": "Grootste manga-website in Vietnam met de laatste update"
 	},
 	"url": [
-		"www.nettruyenmoi.com",
-		"nettruyenmoi.com"
+		"www.nettruyenco.com",
+		"nettruyenco.com"
```
Comment (Slowlife01) [inline websites/N/Nettruyen/dist/metadata.json:15]: This redirect to `www.nettruyenco.com`? If so there's no need to include this

**Review by Slowlife01 — CHANGES_REQUESTED** [review]
Might want to use RegExp at some point

**Comment by ImRodry** [comment]
Please address the review in the coming days or this PR will be closed

**Comment by EncryptedDev** [comment]
Closing due to inactivity

**Pull request closed without merging**